tests/utils/__init___test.py

```diff
@@ -1,10 +1,13 @@
+# type: ignore
```

Comment:
Why is this needed?

Reply:
This file contains 155 pyright errors at the moment, unrelated to the changes I'm doing, so I set the "ignore" at the begining of the file to avoid repeating it on each affected line.

But now that I think on this twice, maybe it makes more sense to no disabling the entire file but only affected lines, so pyright checks will continue running on future changes for this file.